--- config.sub.orig	2010-12-07 16:42:10.000000000 +0100
+++ config.sub	2010-01-15 14:18:17.000000000 +0100
@@ -280,6 +280,8 @@
 	| mn10200 | mn10300 \
 	| moxie \
 	| mt \
+        | mtsparc \
+        | mtalpha \
 	| msp430 \
 	| nios | nios2 \
 	| ns16k | ns32k \
@@ -366,6 +368,8 @@
 	| mipstx39-* | mipstx39el-* \
 	| mmix-* \
 	| mt-* \
+	| mtalpha-* \
+	| mtsparc-* \
 	| msp430-* \
 	| nios-* | nios2-* \
 	| none-* | np1-* | ns16k-* | ns32k-* \
